test/css/all.js

```diff
@@ -0,0 +1,25 @@
+const assert = require('assert').strict;
+
+const css = require('@webref/css');
+
+describe('@webidl/css module', () => {
+  it('listAll', async () => {
+    const files = await css.listAll();
+    assert(Object.keys(files).length > 0);
+    for (const [shortname, file] of Object.entries(files)) {
+      assert.equal(shortname, file.shortname);
+      assert(/^[a-z0-9-_]+$/i.exec(shortname),
+          `invalid shortname: ${shortname}`);
+    }
+  });
+
+  it('parseAll', async () => {
+    const all = await css.parseAll();
+    assert(Object.keys(all).length > 0);
+    for (const desc of Object.values(all)) {
+      assert(desc);
+      assert(desc.spec);
```

Comment:
I spotted this in the output as well. Since it's data from browser-specs, and the same convenience isn't included in `@webref/idl`, should we actually hide this and not make it part of the API? If it's needed, it would be nice to have a common way of getting this extra information for both `@webref/css` and `@webref/idl`.

Reply:
I note that, in a way, the same info appears in `@webref/idl`, it's just hidden in comments because we're using a text format, e.g.:
```
// Source: Accelerometer (https://w3c.github.io/accelerometer/)
```

I would keep the info there, mostly because it's useful when you manually browse one of these files and want to understand where the info comes from and/or check the source spec.

It could still be useful to have an `index.js` file in the browser-specs package that exports a function that takes a series or spec shortname as input and returns the current specification that matches the shortname. Or did you have something else in mind?

Reply:
A util like that in browser-specs is what I had in mind. But let's not dabble with that now. We can release 2.0.0 any time we want to change things.

Reply:
Filed https://github.com/w3c/browser-specs/issues/256.